BoxContentSDK/BoxContentSDKTests/BOXFileTests.m

```diff
@@ -176,8 +176,88 @@ - (void)test_that_file_with_all_fields_is_parsed_correctly_from_json
     for (NSUInteger i = 0; i < file.collections.count ; i++) {
         [self assertModel:file.collections[i] isEquivalentTo:expectedCollections[i]];
     }
+    
+    XCTAssertTrue(file.isFile);
+    XCTAssertFalse(file.isFolder);
+    XCTAssertFalse(file.isBookmark);
+}
 
-#ifdef BOX_REPRESENTATIONS_ENDPOINT_TESTS    
+- (void)test_that_file_with_all_fields_with_representations_is_parsed_correctly_from_json
+{
+    // TODO: We need to revisit file_all_fields to make sure it reflects the latest server API doc.
+    NSDictionary *dictionary = [self dictionaryFromCannedJSON:@"file_all_fields_representations"];
+    BOXFile *file = [[BOXFile alloc] initWithJSON:dictionary];
+    
+    XCTAssertEqualObjects(@"123", file.modelID);
+    XCTAssertEqualObjects(@"Halloweenie-3.jpg", file.name);
+    XCTAssertEqualObjects(@"0", file.etag);
+    XCTAssertNil(file.sequenceID);
+    XCTAssertEqualObjects(@"", file.itemDescription);
+    
+    NSNumber *fileSize = [NSNumber numberWithInteger:622592];
+    XCTAssertEqualObjects(fileSize, file.size);
```

Comment:
I think we could jut write this as `XCTAssertEqualObjects(@622592, file.size);`, since it's only used once.